service/policy/db/migrations/20240821000000_add_res_map_groups_metadata_field.sql

```diff
@@ -0,0 +1,13 @@
+-- +goose Up
+-- +goose StatementBegin
+
+ALTER TABLE resource_mapping_groups ADD COLUMN metadata JSONB;
```

Comment:
Can we please add an existence check to each?

```suggestion
ALTER TABLE IF EXISTS resource_mapping_groups ADD COLUMN metadata JSONB;
```

Reply:
Done